mcp/kubev2v/kubevirt_server.py

```
@@ -0,0 +1,2146 @@
+        try:
+            # Try kubectl first for volume listing
+            result = subprocess.run(
+                ["kubectl"] + args,
+                capture_output=True,
+                text=True,
+                check=True,
+                timeout=120
+            )
+            volumes = result.stdout.strip().split()
+            return f"Volumes attached to VM '{vm_name}':\n" + "\n".join(f"- {vol}" for vol in volumes if vol)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)